setup.py

```diff
@@ -51,6 +51,8 @@
           "Programming Language :: Python :: 3.4",
```

Comment:
On line 41, if going to v1.0, then may want to change this:  `Development Status :: 4 - Beta`

Reply:
Good point, done.